Update Docs API Digital Certificates & Domains (#64)
aziontech/docs_en: api/v3/domains/index.md

Through the Digital Certificates API you can consult, create, remove or update Domains used by Edge Applications

> 1. [Consult list of domains](#consultar-lista-de-domains)

> > 2. [Consults data from a domain id](#consultar-dados-de-um-domains-id)

Reviewer: ```suggestion
> 2. [Consults data from a domain id](#consult-domains-id)
```

> > 3. [Consults data from a domain name](#consultar-dados-de-um-domains-name)

Reviewer: ```suggestion
> 3. [Consults data from a domain name](#consult-domains-name)
```

> > 4. [Delete a domain](#deletar-um-domains)

Reviewer: ```suggestion
> 4. [Delete a domain](#delete-domain)
```

> > 5. [Create a new domain](#criar-um-novo-domains)

Reviewer: ```suggestion
> 5. [Create a new domain](#create-domain)
```

> > 6. [Overwrite a domain](#sobrescrever-um-domains)

Reviewer: ```suggestion
> 6. [Overwrite a domain](#overwrite-domain)
```

> > 7. [Update fields in a domain](#atualizar-campos-de-um-domains)

Reviewer: ```suggestion
> 7. [Update fields in a domain](#update-fields-domain)
```
